java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -3246,13 +3246,12 @@ public final Table extractRe(String pattern) throws CudfException {
    * @link https://docs.rapids.ai/api/libcudf/nightly/md_regex.html
    
    * @param pattern The regex pattern
-   * @param idx The regex group index (only 0 is supported currently). 
+   * @param idx The regex group index. 
```

Comment:
Just small nit to make this consistent with other lines.
```suggestion
   * @param idx The regex group index
```